wiki/system-design-development/electronic-and-actuator-selection.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Title goes here
+published: true
+---
+## Summary
+DC Motor Selection - Link 1
```

Comment:
This isn't our format. Please see [the template](https://github.com/RoboticsKnowledgebase/roboticsknowledgebase.github.io/blob/master/_templates/template.md) for additional instructions.